sensor_msgs/Image image
sensor_msgs/RegionOfInterest roi
geometry_msgs/Point[] shape
---
string                face_id
